grafana/dashboards/kubernetes.json

```diff
@@ -1000,10 +1000,10 @@
       "steppedLine": true,
       "targets": [
         {
-          "expr": "sum (rate (container_cpu_usage_seconds_total{image!=\"\",name=~\"^k8s_.*\",kubernetes_io_hostname=~\"^$Node$\"}[1m])) by (pod_name)",
+          "expr": "sum (rate (container_cpu_usage_seconds_total{image!=\"\",kubernetes_io_hostname=~\"^$Node$\"}[1m])) by (pod_name, pod)",
           "interval": "10s",
           "intervalFactor": 1,
-          "legendFormat": "{{ pod_name }}",
+          "legendFormat": "{{ pod_name }}{{ pod }}",
```

Comment:
Is `pod_name` still a correct label? because all I can see in the dashboard is it being statically printed.
![2020-09-29_14-50](https://user-images.githubusercontent.com/7892868/94539689-33dbe900-0263-11eb-9cf0-ea23651af179.png)



Should we just remove it? as I see `{{ pod }}` is added everywhere it is present?

Reply:
As suggested by alpeb in #4393 I've letted `pod` and `pod_name` in order to keep things compatible.
Can you tell me what version did you used ? Have you used kind or minikube or ... ?

Thank's for you answers

Reply:
@aimbot31 Ahh, That makes sense! I tried this out in kind.

Do these `json` files follow the same go templating? (They kinda look like it?) If yes? maybe we can have some kind of check to print only the `pod` when `pod_name` is empty?

cc: @alpeb

Reply:
@Pothulapati That's weird since  `pod_name` should just be empty... Although I did try after merging `main` that brings a new version of Grafana. @aimbot31 could you please merge `main` to your branch so we can give it another test?

Reply:
Yup i don't understand this because i'm not seeing pod or pod_name, just the only value filled.

Reply:
@alpeb Done

Reply:
Awesome, Works now. and I don't see `pod_name` anymore.